src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/AvailableLocationsService.kt

```diff
@@ -70,9 +75,16 @@ class AvailableLocationsService(
       }
     }
 
-    return AvailableLocationsResponse(availableLocations).also { log.info("AVAILABLE LOCATIONS: found ${it.locations.size} available locations matching request $request") }
+    return AvailableLocationsResponse(
+      availableLocations
+        .sortedWith(compareBy({ it.startTime }, { it.name }))
```

Comment:
Basic sorting here @smcveigh941 it also reduces the number of locations if there are duplicate times.

Reply:
We might have to prioritise which of a same-time rooms is the most suitable, but looks good for now.

Reply:
Indeed, on my radar just not exactly sure how that will work (yet).